apistructs/pipeline_report.go

```diff
@@ -58,67 +55,7 @@ var (
 // PipelineReportMeta 流水线报告元数据，前端根据该数据拼装报告详情界面
 type PipelineReportMeta map[string]interface{}
 
-// PipelineReportCreateRequest 报告创建请求
-type PipelineReportCreateRequest struct {
-	PipelineID uint64             `json:"pipelineID"`
-	Type       PipelineReportType `json:"type"`
-	Meta       PipelineReportMeta `json:"meta"`
-
-	IdentityInfo
-}
-
-func (req PipelineReportCreateRequest) BasicValidate() error {
-	if req.PipelineID == 0 {
-		return fmt.Errorf("missing pipelineID")
-	}
-	if err := strutil.Validate(string(req.Type), strutil.MinLenValidator(1), strutil.MaxLenValidator(32)); err != nil {
-		return fmt.Errorf("invalid type: %v", err)
-	}
-	return nil
-}
-
-type PipelineReportCreateResponse struct {
-	Header
-	Data *PipelineReport `json:"data"`
-}
-
 type PipelineReportSetGetResponse struct {
 	Header
 	Data *PipelineReportSet `json:"data"`
 }
-
-type PipelineReportSetPagingRequest struct {
-	PipelineIDs []uint64             `schema:"-"`
-	Sources     []PipelineSource     `schema:"source"`
-	Types       []PipelineReportType `schema:"type"`
-
-	/////////////////////////
-	// pipeline 分页查询参数 //
-	/////////////////////////
-	// labels
-	// &mustMatchLabel=key2=value3
-	MustMatchLabelsQueryParams []string `schema:"mustMatchLabel"`
```

Comment:
how to handle `schema` flag

Reply:
![image](https://user-images.githubusercontent.com/30427474/177682630-524b16bd-5e7a-4557-8c6f-7795d2a12d7d.png)
![image](https://user-images.githubusercontent.com/30427474/177682799-5695b69a-0c69-4082-9492-6ba09450e34d.png)
`PageListPipelines` func will handle `schema` flag